src/exts/tag.py

```diff
@@ -101,57 +102,35 @@ async def list(self, ctx: interactions.CommandContext):
 
         log.debug("Matched for list. Returning result...")
 
-        embed = interactions.Embed(
-            title="Tags list",
-            description="This is the current list of existing tags.",
-            color=0x5865F2,
-            fields=[
-                interactions.EmbedField(
-                    name="Names",
-                    value=" ",
-                    inline=True,
-                ),
-                interactions.EmbedField(
-                    name="(cont.)",
-                    value=" ",
-                    inline=True,
-                ),
-            ],
-        )
-        _last_name: str = ""
         _id: int = 0
+        _pages: list = []
+        _contents: list = []
+        _embeds: list = []
+        _value: str = ""
 
-        for tag in db:
-            if (len(tag) + len(embed.fields[0].value)) < 900:
-                embed.fields[0].value += f"` {_id} ` {tag}"
-            else:
-                break
+        def _divide(_list):
+            for i in range(0, len(_list), 5):
+                yield _list[i : i + 5]
 
-            _last_name = tag
+        for tag in db:
+            _value += f"` {_id} ` {tag}\n"
             _id += 1
 
-        if _id < len(db):
-            ...
+        for _v in _divide(_value.split("\n")):
+            _contents.append(_v)
 
-        paginator = interactions.ActionRow(
-            components=[
-                interactions.Button(
-                    style=interactions.ButtonStyle.PRIMARY,
-                    label="Previous",
-                    emoji=interactions.Emoji(id=None, name="⬅️", animated=False),
-                    custom_id="list_navigate_left",
-                ),
-                interactions.Button(
-                    style=interactions.ButtonStyle.PRIMARY,
-                    label="Next",
-                    emoji=interactions.Emoji(id=None, name="➡️", animated=False),
-                    custom_id="list_navigate_right",
-                    disabled=True if len(embed.fields[0].value) <= 900 else False,
-                ),
-            ]
-        )
+        for _c in _contents:
+            new_embed = interactions.Embed(
+                title="Tag list",
+                description="This is the list of currently existing tags.",
+                color=0x5865F2,
+            )
+            new_embed.add_field(name="Names", value="\n".join(_c))
+            _embeds.append(new_embed)
 
-        await ctx.send(embeds=embed, components=paginator)
+        await Paginator(
+            client=self.client, ctx=ctx, pages=[Page(embeds=embed) for embed in _embeds]
+        ).run()
```

Comment:
```suggestion
        paginator = Paginator(
            client=self.client, ctx=ctx, pages=[Page(embeds=embed) for embed in _embeds], 
							timeout=600,
        )

        await paginator.run()
```
10 minutes timeout is fine ig

Reply:
I think 10 mins is too much.

Reply:
Reduced to 5